plasmapy/diagnostics/thomson.py

```diff
@@ -341,6 +345,11 @@ def spectral_density(
         Defaults to [0, 1, 0] which, along with the default ``probe_vec``,
         corresponds to a 90° scattering angle geometry.
 
+    scattering_angle : float
```

Comment:
You could phrase this as `either scattering_angle` or both `probe_vec` and `scatter_vec` keywords must be provided.`